CMakeLists.txt

```diff
@@ -0,0 +1,137 @@
+cmake_minimum_required(VERSION 3.1)
+project(libdill VERSION 1.6 LANGUAGES C)
+
+include(CheckSymbolExists)
+include(CheckFunctionExists)
+
+file(GLOB sources *.c dns/*.c)
```

Comment:
Do not use `file(GLOB)` here. From [documentation](https://cmake.org/cmake/help/latest/command/file.html):
```
We do not recommend using GLOB to collect a list of source files from your
source tree. If no CMakeLists.txt file changes when a source is added or
removed then the generated build system cannot know when to ask CMake to
regenerate.
```

Reply:
OK